app/src/main/java/org/zalando/nakadi/config/NakadiConfig.java

```diff
@@ -68,9 +71,25 @@ public List<Check> prePublishingChecks(final EventOwnerExtractorFactory eventOwn
         return Lists.newArrayList(
                 new EventTypeCheck(),
                 new EventOwnerSelectorCheck(eventOwnerExtractorFactory, authValidator),
+                new EventSizeCheck(),
                 partitioningCheck,
                 enrichmentCheck,
                 new EventKeyCheck()
         );
     }
+
+    @Bean
+    @Qualifier("pre-deleting-checks")
+    public List<Check> preDeletingChecks(final EventOwnerExtractorFactory eventOwnerExtractorFactory,
+                                         final AuthorizationValidator authValidator,
+                                         final PartitioningCheck partitioningCheck) {
+        return Lists.newArrayList(
+                new EventTypeCheck(),
+                new EventOwnerSelectorCheck(eventOwnerExtractorFactory, authValidator),
+                new EventDeletionCleanupPolicyCheck(),
+                new EventDeletionEmptyPayloadCheck(),
+                partitioningCheck,
+                new EventKeyCheck()
```

Comment:
This check seems to do nothing, should we remove it?

Reply:
This is fixed now.